v2/azure-arm.yaml

```diff
@@ -2554,6 +2598,9 @@ objectModelConfiguration:
       Databricks:
         ResourceId:
           $armReference: true
+      DatabricksProperties:
+        DatabricksAccessToken:
```

Comment:
Breaking change?

Reply:
Yup.

Reply:
Where are the associated code-generated changes for this?

I only see 2 files in this PR now?

Reply:
This isn't new - it's relocated from just below - we're supposed to have things in alphabetical order, so I moved it up into place. That's why there's no change to generated code.